magicctapipe/scripts/lst1_magic/magic_calib_to_dl1.py

```diff
@@ -380,6 +379,8 @@ def main():
         help="Max. number of processed showers",
     )
 
+
```

Comment:
you can also drop those 2 empty lines that were added

Reply:
Done